mikochiku_alarm.py

```diff
@@ -116,24 +117,32 @@ def set_target_channel(self, qmode8ndex):
         self.search_ch_id = member['channel_id']
 
     def check_live(self):
+        videos = []
         buff_video_id_set = self.get_live_video_id(self.search_ch_id)
-        for getting_video_id in buff_video_id_set:
+        for getting_video_id in buff_video_id_set.keys():
             if getting_video_id in self.old_video_id_list:
-                return
+                continue
+            videos.append(
+                {'vid': getting_video_id,
+                 'title': buff_video_id_set[getting_video_id]})
             self.old_video_id_list.append(getting_video_id)
             if len(self.old_video_id_list) > 30:
                 self.old_video_id_list.pop(0)
             log.info(self.localized_text("started"))
-            log.debug(f"buff_video_id_set: {buff_video_id_set}")
+            log.debug(f"buff_video_id_set: {[id for id in buff_video_id_set.keys()]}")
             log.debug(f"self.old_video_id_list {self.old_video_id_list}")
             self.alarm_stop.click()
             self.alarm_state = "stop"
             self.alarm_stop.setText(self.localized_text("stop"))
-            if self.webbrowser_cb.checkState():
+            should_open_browser = self.webbrowser_cb.checkState()
+            if should_open_browser:
                 webbrowser.open(
                     "https://www.youtube.com/watch?v=" + getting_video_id)
             if self.alarm_cb.checkState():
                 self.alarm_sound()
+        if len(videos) > 0:
+            t = toast.Toast(self, videos, should_open_browser)

```

Comment:
`should_open_browser` might be referenced before assignment

Reply:
Oh, i missed it. Fixed.